<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2193</width>
    <height>1130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout" stretch="1,15">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="topNavBar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>68</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> #topNavBar {
	background:white;
    border-bottom: 2px solid #ccc;
}</string>
        </property>
        <widget class="QPushButton" name="hideSideBarBtn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>57</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>57</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: transparent;
		border-radius: 10px;
}
QPushButton:hover {
    background-color: #F5F5F5;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="funresources.qrc">
           <normaloff>:/images/assets/panel-left-close.png</normaloff>:/images/assets/panel-left-close.png</iconset>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>20</y>
           <width>161</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="companyLogo" stretch="1,2,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="funresources.qrc">:/images/assets/swords.png</pixmap>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:green</string>
            </property>
            <property name="text">
             <string>Bravora</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>2070</x>
           <y>11</y>
           <width>116</width>
           <height>52</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="accountBtn">
            <property name="minimumSize">
             <size>
              <width>57</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
        background-color: transparent;
		border-radius: 10px;
}
QPushButton:hover {
    background-color: #F5F5F5;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="funresources.qrc">
              <normaloff>:/images/assets/circle-user.png</normaloff>:/images/assets/circle-user.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="settingsBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>57</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
        background-color: transparent;
		border-radius: 10px;
}
QPushButton:hover {
    background-color: #F5F5F5;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="funresources.qrc">
              <normaloff>:/images/assets/settings.png</normaloff>:/images/assets/settings.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayoutDown" stretch="1,8">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="leftSideBar" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">#leftSideBar{
    border-right: 2px solid #ccc;
background:white;
}


</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>5</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="leftButtonsLayout" stretch="0,0,0,0,0,0,0,0,0">
             <property name="spacing">
              <number>-1</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QPushButton" name="homeBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border-radius: 5px;
	text-align: left;
	padding-left: 5px; 
}

QPushButton:hover {
    background-color: #F5F5F5;
}</string>
               </property>
               <property name="text">
                <string>Home</string>
               </property>
               <property name="icon">
                <iconset resource="funresources.qrc">
                 <normaloff>:/images/assets/house.png</normaloff>:/images/assets/house.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="progBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border-radius: 5px;
	text-align: left;
	padding-left: 5px; 
}

QPushButton:hover {
    background-color: #F5F5F5;
}</string>
               </property>
               <property name="text">
                <string>Programming</string>
               </property>
               <property name="icon">
                <iconset resource="funresources.qrc">
                 <normaloff>:/images/assets/gamepad-2.png</normaloff>:/images/assets/gamepad-2.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="artBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border-radius: 5px;
	text-align: left;
	padding-left: 5px; 
}

QPushButton:hover {
    background-color: #F5F5F5;
}</string>
               </property>
               <property name="text">
                <string>Art</string>
               </property>
               <property name="icon">
                <iconset resource="funresources.qrc">
                 <normaloff>:/images/assets/brush.png</normaloff>:/images/assets/brush.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pmBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border-radius: 5px;
	text-align: left;
	padding-left: 5px; 
}

QPushButton:hover {
    background-color: #F5F5F5;
}</string>
               </property>
               <property name="text">
                <string>Product</string>
               </property>
               <property name="icon">
                <iconset resource="funresources.qrc">
                 <normaloff>:/images/assets/notebook.png</normaloff>:/images/assets/notebook.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Policy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="beBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
    background-color: white;
    color: black;
    border-radius: 5px;
	text-align: left;
	padding-left: 5px; 
}

QPushButton:hover {
    background-color: #F5F5F5;
}</string>
               </property>
               <property name="text">
                <string>Backend</string>
               </property>
               <property name="icon">
                <iconset resource="funresources.qrc">
                 <normaloff>:/images/assets/code-xml.png</normaloff>:/images/assets/code-xml.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="pageStack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">#pageStack{
background:red;
  border-left: 2px solid #ccc;
}
</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="programmingPage">
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Programming Panel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="artPage">
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Game Art Panel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="homePage">
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Home</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="backendPage">
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Backend  Panel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="productPage">
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>250</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Product  Panel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="funresources.qrc"/>
 </resources>
 <connections/>
</ui>
